src/main/java/com/iotiq/user/internal/UserService.java

```diff
@@ -79,6 +83,23 @@ public User update(UUID id, UserUpdateDto request) {
 
         return user;
     }
+    public void validateUniqueUserData(String username, String email) {
+        if (userRepository.existsByAccountInfoUsername(username)) {
+            throw new DuplicateUserDataException("username");
+        }
+        if (userRepository.existsByPersonalInfoEmail(email)) {
+            throw new DuplicateUserDataException("email");
+        }
```

Comment:
Can these two repository methods be combined into something like `existsByAccountInfoUsernameOrPersonalInfoEmail`? I don't know which is better